cms/djangoapps/api/v1/serializers/course_runs.py

```diff
@@ -181,8 +195,27 @@ def update(self, instance, validated_data):
             'display_name': instance.display_name
         }
         fields.update(validated_data)
-        new_course_run_key = rerun_course(user, course_run_key, course_run_key.org, course_run_key.course, _id['run'],
-                                          fields, background=False)
+        if 'MITx+7.00x' in unicode(course_run_key):
+            new_course_run_key = _execute_method_and_log_time(
```

Comment:
More simplier if you move the `if` condition inside `_execute_method_and_log_time` method. 
```
def _execute_method_and_log_time(func, *args):
        """
        Call func passed in method with logging the time it took to complete.
        Temporarily added for EDUCATOR-4013, we will remove this once we get the required information.
        """
        if not 'MITx+7.00x' in unicode(course_run_key):
            return func(*args)
        start_time = time.time()
        output = func(*args)
        log.info(u'[%s] [%s] completed in [%f]', func.__name__, args[1], (time.time() - start_time))
        return output



_execute_method_and_log_time(
    rerun_course,
    user,
    course_run_key,
    course_run_key.org,
    course_run_key.course,
    _id['run'],
    fields,
    False
)
```